Src/Sarif.PatternMatcher/ValidatorsCache.cs

```diff
@@ -277,10 +309,19 @@ public ValidatorsCache(IEnumerable<string> validatorBinaryPaths = null, IFileSys
                             isValidDynamic = null;
                         }
 
-                        ruleToMethodMap[typeName] = new ValidationMethodPair
+                        MethodInfo disableValidationCaching = type.BaseType.GetMethod(
```

Comment:
>GetMethod [](start = 76, length = 9)

Let's use a property for this. It's fine to stick with 'ValidationMethods' as a name. You can retrieve the setter for the property and store it there.